com.unity.render-pipelines.universal/Runtime/UniversalRendererData.cs

```diff
@@ -236,19 +235,6 @@ internal TileSize tileSize
             }
         }
 
-        /// <summary>
-        /// Controls when URP renders via an intermediate texture.
-        /// </summary>
-        public IntermediateTextureMode intermediateTextureMode
```

Comment:
hm, This will probably fail API validation.

Reply:
Hmm, is there something we can do? Leaving in the API point but unfunctional would be odd.

Reply:
We have two options, either disable API validation or make it obsolete right away. None of them are good. I vote for making it obsolete as it's less disruptive.

For master / this PR it's fine to remove it.